tsconfig.json

```diff
@@ -27,6 +31,14 @@
       }
     ]
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
-  "exclude": ["node_modules"]
+  "include": [
+    "next-env.d.ts",
+    "**/*.ts",
+    "**/*.tsx",
+    ".next/types/**/*.ts",
+    "/Users/ronnyr/Projects/wix-commerce-ticketing-nextjs-template/.next/types/**/*.ts"
```

Comment:
@ronnyrin Hey this is probably not intentional.

Reply:
lol